application/functest/fund_test.go

```diff
@@ -91,54 +91,6 @@ func checkBalanceAndDepositFewTimes(t *testing.T, m *launchnet.User, expectedBal
 		depositStr, expectedDeposit)
 }
 
-func TestFundsTransferDeposit(t *testing.T) {
```

Comment:
add check balance tests on new accounts

Reply:
we have already this test and I adjust it. Take closer look.